core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -488,9 +488,16 @@ private[spark] class ExecutorAllocationManager(
     newExecutorTotal = numExistingExecutors
     if (testing || executorsRemoved.nonEmpty) {
       executorsRemoved.foreach { removedExecutorId =>
+        // If it is cachedBlcok timeout is configured using
+        // spark.dynamicAllocation.cachedExecutorIdleTimeout
+        val idleTimeout = if (SparkEnv.get.blockManager.master.hasCachedBlocks(removedExecutorId)) {
```

Comment:
`blockManagerMaster.hasCachedBlocks(removedExecutorId)`?

Reply:
I have updated the code as per comments